docs/en/connector-v2/sink/StarRocks.md

```diff
@@ -20,31 +20,68 @@ The internal implementation of StarRocks sink connector is cached and imported b
 
 ## Sink Options
```

Comment:
I think an example needs to be added to the documentation

Reply:
added